docs/contracts.rst

```diff
@@ -40,7 +40,7 @@ This means that cyclic creation dependencies are impossible.
 
 ::
 
-    pragma solidity ^0.4.16;
+    pragma solidity ^0.4.21;
```

Comment:
Please don't use `0.4.21` as the tests will fail. Use a comment instead and update it in #3634.